go/websocketbridge/bin/spec_ci

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+
+bin/spec
```

Comment:
yeah nvm with these. You still keep these files with with my changes. The only change is that `vendor` folder and `glide.yaml` are now in the top level so you won't need a local vendor folder anymore if you already have one.